### DISCUSSION TITLE
Does libdatachannel-sfu support WebRTC-TWCC?

### DISCUSSION THREAD

**Discussion by lokenetwork** [paullouisageneau/libdatachannel#1057] [Q&A]
I search TWCC in the code,but I not found

**Comment by paullouisageneau** [comment]
The extension is not supported for now, but [Vagon has an implementation in their fork](https://github.com/vagonhq/libdatachannel/blob/twcc-streamer/examples/streamer/main.cpp) by @ognkrmms and @kuzux.
